extractor/src/test/java/org/schabi/newpipe/extractor/services/bandcamp/BandcampStreamExtractorTest.java

```diff
@@ -0,0 +1,100 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.extractor.services.bandcamp;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.schabi.newpipe.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.services.bandcamp.extractors.BandcampExtractorHelper;
+import org.schabi.newpipe.extractor.services.bandcamp.extractors.BandcampStreamExtractor;
+import org.schabi.newpipe.extractor.stream.StreamExtractor;
+
+import java.io.IOException;
+import java.util.Calendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.schabi.newpipe.extractor.ServiceList.Bandcamp;
+
+public class BandcampStreamExtractorTest {
+
+    private static BandcampStreamExtractor extractor;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        NewPipe.init(DownloaderTestImpl.getInstance());
+        extractor = (BandcampStreamExtractor) Bandcamp
+                .getStreamExtractor("https://zachbenson.bandcamp.com/track/kitchen");
+        extractor.fetchPage();
+    }
+
+    @Test
+    public void testServiceId() {
+    }
+
+    @Test
+    public void testName() throws ParsingException {
+        assertEquals("kitchen", extractor.getName());
+    }
+
+    @Test
+    public void testUrl() throws ParsingException {
+        assertEquals("https://zachbenson.bandcamp.com/track/kitchen", extractor.getUrl());
+    }
+
+    @Test
+    public void testArtistUrl() throws ParsingException {
+        assertEquals("https://zachbenson.bandcamp.com/", extractor.getUploaderUrl());
+    }
+
+    @Test
+    public void testDescription() {
+        assertEquals(831, extractor.getDescription().getContent().length());
```

Comment:
you should not do this, the content can change and 1 character less or more will break the change. You should test with Description.getContent().contains("Something that will probably never be edited");